scripts/single_component_inference.py

```diff
@@ -16,9 +16,9 @@
 
 from jsonargparse import CLI
 
-from arc_spice.data.multieurlex_utils import load_multieurlex_for_translation
+from arc_spice.data.multieurlex_utils import load_multieurlex_for_pipeline
```

Comment:
sorry! my fault for name change and not refactoring properly!

Reply:
not a problem!